tests/unit/utils.ts

```diff
@@ -90,9 +92,14 @@ export function generateCompletionData(
   tstText: string,
   position: CompletionPosition,
   triggerCharacter: string | undefined,
-  envName?: string
+  optParameters?: {
+    lineSoFar?: string;
+    cppTest?: boolean;
+    envName?: string;
+  }
 ) {
-  envName ||= "vcast";
+  // OptParams can be undefined
+  const envName = optParameters?.envName ?? "vcast";
   const languageId = "VectorCAST Test Script";
```

Comment:
I think if this is a coded test, we should be setting the `languageId` to be `cpp` (but we should confirm actually what it should be).

Reply:
Put in cpp as languageId in case of coded tests in 770736c06264d30fd8e6d29f6695849bfad10960 .